java/arcs/android/crdt/RawEntityProto.kt

```diff
@@ -0,0 +1,45 @@
+package arcs.android.crdt
+
+import android.os.Parcel
+import arcs.android.util.readProto
+import arcs.core.data.RawEntity
+
+/** Constructs a [RawEntity] from the given [RawEntityProto]. */
+fun fromProto(proto: RawEntityProto): RawEntity {
```

Comment:
Also - would be nice for this to be an Extension Function on `RawEntityProto`.

Reply:
Done